node-tests/blueprints/route-test.js

```diff
@@ -480,516 +474,17 @@ describe('Blueprint: route', function () {
-            .to.contain("this.route('foo', {")
-            .to.contain("path: ':foo_id/show'")
-            .to.contain('});')
-        )
-
-        .then(() => emberDestroy(['route', 'foo', '--pod', '--path=:foo_id/show']))
-        .then(() =>
-          expect(file('app/router.js'))
-            .to.not.contain("this.route('foo', {")
-            .to.not.contain("path: ':foo_id/show'")
-        );
-    });
-
-    it('route index --pod', function () {
-      return emberGenerate(['route', 'index', '--pod']).then(() =>
-        expect(file('app/router.js')).to.not.contain("this.route('index')")
-      );
-    });
-
-    it('route application --pod', function () {
-      return emberGenerate(['route', 'application', '--pod'])
-        .then(() => expect(file('app/application/route.js')).to.exist)
-        .then(() => expect(file('app/application/template.hbs')).to.exist)
-        .then(() => expect(file('app/router.js')).to.not.contain("this.route('application')"));
-    });
-
-    it('route basic --pod', function () {
-      return emberGenerateDestroy(['route', 'basic', '--pod'], (_file) => {
-        expect(_file('app/basic/route.js')).to.exist;
-        expect(file('app/router.js')).to.not.contain("this.route('index')");
-      });
-    });
-
-    describe('with podModulePrefix', function () {
-      beforeEach(function () {
-        setupPodConfig({ podModulePrefix: true });
-      });
-
-      it('route foo --pod', function () {
-        return emberGenerateDestroy(['route', 'foo', '--pod'], (_file) => {
-          expect(_file('app/pods/foo/route.js')).to.equal(fixture('route/native-route.js'));
-
-          expect(_file('app/pods/foo/template.hbs')).to.equal('{{outlet}}');
-
-          expect(_file('tests/unit/pods/foo/route-test.js')).to.equal(
-            fixture('route-test/default.js')
-          );
-
-          expect(file('app/router.js')).to.contain("this.route('foo')");
-        }).then(() => {
-          expect(file('app/router.js')).to.not.contain("this.route('foo')");
-        });
-      });
-
-      it('route foo.js --pod', function () {
-        return emberGenerateDestroy(['route', 'foo.js', '--pod'], (_file) => {
-          expect(_file('app/pods/foo.js/route.js')).to.not.exist;
-          expect(_file('app/pods/foo.js/template.hbs')).to.not.exist;
-          expect(_file('tests/unit/pods/foo.js/route-test.js')).to.not.exist;
-
-          expect(_file('app/pods/foo/route.js')).to.equal(fixture('route/native-route.js'));
-
-          expect(_file('app/pods/foo/template.hbs')).to.equal('{{outlet}}');
-
-          expect(_file('tests/unit/pods/foo/route-test.js')).to.equal(
-            fixture('route-test/default.js')
-          );
-
-          expect(file('app/router.js')).to.contain("this.route('foo')");
-          expect(file('app/router.js')).to.not.contain("this.route('foo.js')");
-        }).then(() => {
-          expect(file('app/router.js')).to.not.contain("this.route('foo')");
-        });
-      });
-    });
```

Comment:
We still need these pods tests

Reply:
they still exist, they're just higher up in the file https://github.com/cafreeman/ember.js/blob/cfreeman/remove-non-octane-blueprint-tests/node-tests/blueprints/route-test.js#L240-L282